src/mcts/node.h

```diff
@@ -209,35 +208,49 @@ class Node {
   std::string DebugString() const;
 
  private:
-  // List of edges.
+  // Turns out the alignment and padding of the node was costing more than 8
+  // bytes on modern compilers. Simply rearranging the members leads to
+  // significant savings on padding. And if rearranging members for alignment,
+  // may as well arrange them in order of size, to ease future changes.
+
+  // First, the 16-byte EdgeList, which is itself 10 real bytes and 6 pad bytes.
+  // TODO: shrink the padding on this somehow?
+  // Maybe even merge it into this class??
```

Comment:
My initial plan was to make EdgeList packed, and take care that it will be with proper alignment in Node, with uint16 right after it.
Then I decided that it's too hacky. I plan to replace pointer to index in EdgePool anyway, will reduce it from 8 to 4 bytes.

Out of EdgeList edgesize is how I initially implemented it for alignment reasons, didn't like it.

Reply:
2 questions
1 do I understand correctly that edge list is just a linked list?
2. if so, could you store 4 values per node to fix alignment and save memory?

Reply:
@oscardssmith The edge list is a pointer to a vector. I've looked it over and I don't see how to make it smaller. Though it does look like there are few ways to make faster.